packages/lab/src/components/Flow/types/flow.ts

```diff
@@ -75,8 +83,14 @@ export type HvFlowNodeParam = {
   value?: string;
 };
 
-export type HvFlowDefaultAction = "delete" | "duplicate";
+export interface HvFlowNodeAction extends HvActionGeneric {
+  callback?: (node: Node) => void;
+}
 
-export type HvFlowDefaultActions = Omit<HvActionGeneric, "id"> & {
-  id: HvFlowDefaultAction;
+export type HvFlowBuiltInAction = "delete" | "duplicate";
+
+export type HvFlowBuiltInActions = Omit<HvFlowNodeAction, "id" | "callback"> & {
```

Comment:
This type doesn't seem to used in the code. We should use it on the `BaseNode` or remove both `HvFlowBuiltInActions` and `HvFlowBuiltInAction`.